### DISCUSSION TITLE
disable access/error.logs

### DISCUSSION THREAD

**Discussion by Tealk** [NginxProxyManager/nginx-proxy-manager#2170] [Q&A]
Hello,

is there a way to disable the proxy-host-x_access.log, proxy-host-x_error.log redirection-host-x_access.log and redirection-host-x_error.log?

**Comment by technotiger** [comment]
Logs can be disabled by removing/commenting out access_log and error_log lines in /data/nginx/proxy_host/x.conf. Similarly for redirection-host.

**Reply by Tealk** [reply]
but this is always overwritten by the webpanel

**Reply by technotiger** [reply]
That makes sense, however it should work until you make changes via webpanel. Another way can be symlinking log files to /dev/null.

It will be best if the option to disable logs is added in the webpanel.
